wallet/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -930,22 +980,12 @@ deriveSafeCopySimple 2 'extension ''WalletStorage_v2
 deriveSafeCopySimple 3 'extension ''WalletStorage_v3
 deriveSafeCopySimple 4 'extension ''WalletStorage
 
--- | Unsafe address conversion for use in migration. This will throw an error if
---   the address cannot be migrated.
-unsafeCIdToAddress :: WebTypes.CId WebTypes.Addr -> Address
-unsafeCIdToAddress cId = case WebTypes.cIdToAddress cId of
-    Left err -> error $ "unsafeCIdToAddress: " <> err
-    Right x  -> x
-
 instance Migrate AddressInfo where
     type MigrateFrom AddressInfo = AddressInfo_v0
     migrate AddressInfo_v0{..} = AddressInfo
         { adiWAddressMeta = cwamToWam _v0_adiCWAddressMeta
         , adiSortingKey = _v0_adiSortingKey
         }
-      where
-        cwamToWam (WebTypes.CWAddressMeta wid accIdx addrIdx cAddr) =
-            WAddressMeta wid accIdx addrIdx $ unsafeCIdToAddress cAddr
```

Comment:
The use of this function can be replaced by `migrate` once the `Migrate instance for `WAddressMeta` is in place.